components/layout_2020/layout_debug.rs

```diff
@@ -32,38 +32,52 @@ macro_rules! layout_debug_scope(
     )
 );
 
+#[derive(Serialize)]
+struct TreeValues {
+    pub box_tree: Value,
+    pub fragment_tree: Value,
```

Comment:
What do you think of keeping already-serialized `String`s here, instead of `Value`s? I don’t really know which is better, it’s just another idea.

Reply:
Since there's no clear benefit of keeping already-serialized `String`s, I'd prefer to keep the `Value`s to make the parsing on the viewer easier and to allow an easier visualization of the JSON files directly on the browser:

Strings:

![Screenshot 2020-02-21 at 16 51 13](https://user-images.githubusercontent.com/480202/75049599-cdbc0600-54ca-11ea-849e-cfd3258251fd.png)

vs Objects:

![Screenshot 2020-02-21 at 16 56 33](https://user-images.githubusercontent.com/480202/75049915-44590380-54cb-11ea-86a8-6cd03c2042e6.png)

Reply:
I meant `String`s in JSON syntax that would be written directly to files. Your "Strings" screenshot involves double-serialization to JSON, which is not what I was suggesting. But anyway, `Value` is fine (and might be easier to combine if you want more than one in a given `.json` file?)